src/p5.dance.js

```diff
@@ -299,6 +299,14 @@ export default class DanceParty {
     return sprite;
   }
 
+  makeNewDanceSpriteGroup(n, costume, layout) {
```

Comment:
Should we put a cap on `n`?

Reply:
I'm capping it in the block by providing a dropdown (currently set to 3-10)

Reply:
We probably want to also cap it here (or in Dance.js `nativeAPI. makeNewDanceSpriteGroup` call).  A user can hand-edit the XML to bypass any dropdown limits.

Reply:
Does it make more sense to cap the total number of sprites across the board instead. Even if we cap this block, you could just run it multiple times (which I would expect is a more common student workaround than editing XML)

Reply:
Or we let it be and see kids enjoy the Star Wars infinite droid effect :)